### PR TITLE
feat(utils): Use colors to indicate bar level

### PR DESCRIPTION
**Pull request by nderjung**
Signed-off-by: Alexander Jung <alex@unikraft.io>

### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: internal/cli/kraft/cloud/utils/print.go [hunk 1]
```diff
@@ -611,15 +611,22 @@ func PrintServiceGroups(ctx context.Context, format string, resp kcclient.Servic
...
 	width := 36
 
 	var ret strings.Builder
 
 	percent := math.Floor(float64(progress) / float64(max) * float64(width))
 
-	ret.WriteString(strings.Repeat("█", int(percent)))
-	ret.WriteString(strings.Repeat("░", width-int(percent)))
+	color := "green"
+	if percent > 30 {
```
Comment (craciunoiuc) [inline internal/cli/kraft/cloud/utils/print.go]: Isn't `30` a bit small

I would do:
1. [0-60) green
2. [60-80) yellow
3. [80-100] red

File: internal/cli/kraft/cloud/utils/print.go [hunk 2]
```diff
@@ -611,15 +611,22 @@ func PrintServiceGroups(ctx context.Context, format string, resp kcclient.Servic
...
+		color = "red"
+	} else if percent > 20 {
+		color = "yellow"
+	}
+
+	ret.WriteString(cs.ColorFromString(color)(strings.Repeat("█", int(percent))))
+	ret.WriteString(cs.ColorFromString(":243")(strings.Repeat(" ", width-int(percent))))
```
Comment (craciunoiuc) [inline internal/cli/kraft/cloud/utils/print.go:629]: what is this? `:243`

**Review by craciunoiuc — CHANGES_REQUESTED** [review]
comments

File: internal/cli/kraft/cloud/utils/print.go (hunk 2 above)
Comment (nderjung) [reply internal/cli/kraft/cloud/utils/print.go:629]: bg is gray

File: internal/cli/kraft/cloud/utils/print.go (hunk 1 above)
Comment (nderjung) [reply internal/cli/kraft/cloud/utils/print.go]: It's out of 36, not 100. 30 represents ~83%

20 is ~55%

File: internal/cli/kraft/cloud/utils/print.go (hunk 2 above)
Comment (craciunoiuc) [reply internal/cli/kraft/cloud/utils/print.go:629]: add a comment and all good

**Review by craciunoiuc — APPROVED** [review]
All good here. Thanks!

Reviewed-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>
Approved-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>

**Pull request merged**